### PR TITLE
SimpleMvvm in .Net core 

### PR DESCRIPTION
**Pull request by snprani**

### CODE REVIEW

**Comment by tonysneed** [comment]
@snprani It looks like the **.gitignore** file you added is not standard. You should update it to match [this one](https://github.com/github/gitignore/blob/master/VisualStudio.gitignore).

After updating the file, stage and commit it. Then push the commit to your remote branch.

```
git add .
git commit -m "Update .gitignore"
git push
```

File: App.xaml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+﻿<Application x:Class="SimpleMvvmSamples.WPF.App"
+             xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+             xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+             xmlns:locators="clr-namespace:SimpleMvvmSamples.WPF.Locators"
```
Comment (tonysneed) [inline App.xaml]: You can remove the use of Locators, since we should use dependency injection instead.

File: App.xaml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+﻿<Application x:Class="SimpleMvvmSamples.WPF.App"
+             xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+             xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+             xmlns:locators="clr-namespace:SimpleMvvmSamples.WPF.Locators"
+             StartupUri="MainWindow.xaml">
+    <Application.Resources>
+        <locators:ViewModelLocator x:Key="Locator" />
```
Comment (tonysneed) [inline App.xaml]: Remove this locator.

File: MainWindow.xaml [hunk 3]
```diff
@@ -0,0 +1,15 @@
+﻿<Window x:Class="SimpleMvvmSamples.WPF.MainWindow"
+        xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+        xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+        xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+        xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+        xmlns:local="clr-namespace:SimpleMvvmSamples.WPF"
+        xmlns:views="clr-namespace:SimpleMvvmSamples.WPF.Views"
+        mc:Ignorable="d"
+        Title="Main Window" Height="300" Width="525"
+        DataContext="{Binding Source={StaticResource Locator}, Path=MainPageViewModel}">
```
Comment (tonysneed) [inline MainWindow.xaml]: Remove the locator.

File: MainWindow.xaml [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+        xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+        xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+        xmlns:local="clr-namespace:SimpleMvvmSamples.WPF"
+        xmlns:views="clr-namespace:SimpleMvvmSamples.WPF.Views"
+        mc:Ignorable="d"
+        Title="Main Window" Height="300" Width="525"
+        DataContext="{Binding Source={StaticResource Locator}, Path=MainPageViewModel}">
+    <StackPanel>
+        <TextBlock Text="{Binding Path=BannerText}" FontFamily="Verdana" FontSize="18" FontWeight="Bold" HorizontalAlignment="Center" Margin="0,0,0,10" />
+        <views:CustomerView />
```
Comment (tonysneed) [inline MainWindow.xaml]: You can comment out CustomerView for now.

**Comment by snprani** [comment]
Removed locator and commented CustomerView

**Comment by tonysneed** [comment]
Thanks. What does the commit message mean, “Added via upload”? How are you adding the file?

**Comment by snprani** [comment]
Updated files

**Comment by tonysneed** [comment]
Why did you close the PR?

**Comment by snprani** [comment]
> Why did you close the PR?

I'm not sure whether to close it or not.

**Comment by tonysneed** [comment]
Usually after submitting a PR, and making any requested changes, you wait for the owner to merge the PR, which will automatically close it.

**Comment by snprani** [comment]
Sure, i was not aware of that actually.

**Pull request closed without merging**